package.json

```diff
@@ -21,7 +21,7 @@
     "format": "prettier --write '**/*.{js,md,scss,ts}' '!**/{build,es,lib,storybook,ts,umd,components}/**'",
     "format:diff": "prettier --list-different '**/*.{js,md,scss,ts}' '!**/{build,es,lib,storybook,ts,umd,components}/**'",
     "format:staged": "prettier --write",
-    "gather-deploy-files": "yarn workspaces run gather-deploy-files",
+    "gather-deploy-files": "yarn workspace carbon-components gather-deploy-files && yarn workspace carbon-components-react gather-deploy-files",
```

Comment:
This can just be `lerna run gather-deploy-files` I believe right?

Reply:
Great suggestion - Was looking for something similar, thanks @vpicone!